os/include/tinyara/mm/mm.h

```diff
@@ -703,6 +703,7 @@ void heapinfo_set_caller_addr(void *address, mmaddress_t caller_retaddr);
 void heapinfo_add_size(struct mm_heap_s *heap, pid_t pid, mmsize_t size);
 void heapinfo_subtract_size(struct mm_heap_s *heap, pid_t pid, mmsize_t size);
 void heapinfo_update_total_size(struct mm_heap_s *heap, mmsize_t size, pid_t pid);
+void heapinfo_update_stack_pid(void *stack_ptr, pid_t pid);
```

Comment:
`heapinfo update stack pid` is difficult to know what it is. How about `heapinfo set stack node`?

Reply:
Changed, Thanks~